src/components/ConditionalFilter/UI/MetadataInput.tsx

```diff
@@ -0,0 +1,73 @@
+import { Box, Input } from "@saleor/macaw-ui-next";
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { metadataInputMessages } from "../intl";
+import { FilterEventEmitter } from "./EventEmitter";
+import { DoubleTextOperator } from "./types";
+
+interface MetadataInputProps {
+  index: number;
+  selected: DoubleTextOperator;
+  emitter: FilterEventEmitter;
+  error: boolean;
+  disabled: boolean;
+}
+
+export const MetadataInput = ({
```

Comment:
suggestion: WDYT about make this component more generic and name it like DoubleTextInput?